### PR TITLE
Fix Misc Engineering Diffraction 2 Bugs

### LINKED ISSUE #28102: Engineering Diffraction Minor Bugs
Unscripted testing has revealed some bugs that should be fixed before release:
- [x] Settings menu could do with a different title to the main window
- [x] Y Axis label on calibration plots is missing a bracket
- [x] Plots not appearing when calibration is cropped
- [x] Test guide needs some updates for clarity. 

Uses suggestions from #28028

### PR DESCRIPTION
**Pull request by cailafinn**
**Description of work.**
- Fixed TOF Peak plots not appearing when cropping after calibration
- Fixed incorrect y axis labels on calibration plots
- Refactored Plotting functions
- Included "Settings" in settings menu window title
- Updated test guide for clarity, particularly with cropping. 

**To test:**
- Follow the test guide for engineering diffraction here: http://developer.mantidproject.org/Testing/EngineeringDiffraction/EngineeringDiffraction2TestGuide.html

Fixes #28102

*This does not require release notes* because **it fixes issues in a new interface**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by DannyHindson** [comment]
Thanks for making the changes. I ran through the tests again and the plots are all generated as required. Just a few minor comments:

a) Here's a screenshot of the comment I made earlier about the help icon not doing anything on the settings screen. The help icon is only present in MantidPlot (took me a while to take a screenshot of the cursor but got there in the end!):

EngineeringSettingsHelpIcon

Not something that must be fixed but thought I'd let you know where I'd seen this

b) in the test script under test 4 you've added a comment saying you get one less plot from the calibration screen when you have cropping active. Please could you add a similar comment regarding the output from the Fitting tab?

c) should the "Full calibration" edit box in the Settings screen play any part in this set of tests? It stayed empty throughout for me. Wasn't sure if any of the calibration actions should populate it

**Comment by ConorMFinn** [comment]
> It stayed empty throughout for me.

This is the expected behaviour. Once the ENGINX scientists have generated a full calibration using new data, it can be added to the testing guide. 

> Please could you add a similar comment regarding the output from the Fitting tab?

Did you mean the focus tab?

**Comment by DannyHindson** [comment]
Thanks. Yes I meant the focus tab

**Comment by giovannidisiena** [comment]
@ConorMFinn @DannyHindson Inheriting from `MantidQt::API::MantidDialog` over `QDialog` should remove the help button. Let me know if you have any trouble and I'll take a look 👍

File: scripts/Engineering/gui/engineering_diffraction/settings/settings_view.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 class SettingsView(QtWidgets.QDialog, Ui_settings):
```
Comment (giovannidisiena) [inline scripts/Engineering/gui/engineering_diffraction/settings/settings_view.py]:
```suggestion
class SettingsView(MantidQt::API::MantidDialog, Ui_settings):
```

Comment (giovannidisiena) [reply scripts/Engineering/gui/engineering_diffraction/settings/settings_view.py]: Might need exposing to python

Comment (ConorMFinn) [reply scripts/Engineering/gui/engineering_diffraction/settings/settings_view.py]: For now, I'm just going to set the flags in the view. The exposure seems unnecessary unless this had to be done in many different places.

**Review by DannyHindson — APPROVED** [review]
All looks good now thanks

**Pull request merged**